cypress/e2e/annual-review/unpublish.feature

```diff
@@ -3,29 +3,31 @@ Feature: Check correct email is sent to provider this is about to be unpublished
   Background: Setup list and provider
     Given A "lawyers" list exists for Eurasia
     And eurasia lawyers are due to begin annual review
-    And a list item has been with the provider for 60 days
+    And a list item has been with the provider for 100 days
 
-  Scenario Outline: Provider is sent a an email the day before they're unpublish
-    When <daysBeforeUnpublish> days before unpublish
+  Scenario: Provider is sent a reminder email the day before they're unpublished
+    When 1 days before unpublish
     And the batch process has run
     And the worker process has run
-    Then the unpublish reminder email for <reminderDays> days is sent to eligible providers
+    Then the unpublish reminder email for 1 days is sent to eligible providers
 
-    Examples:
-      | daysBeforeUnpublish | reminderDays |
-      | 41                  | 1            |
-      | 42                  | 0            |
-      | 43                  | -1           |
-
-
-  Scenario: Provider is NOT sent an email if entry is edited before unpublish date
-    And 42 days before unpublish
+  Scenario: The provider is not sent a reminder email on the day they're unpublished
```

Comment:
Out of curiosity why was the examples table removed? Are you confident that those scenarios have been tested with the first test?